src/main/style/checkstyle.xml

```diff
@@ -30,7 +30,7 @@
     </module>
 
     <module name="LineLength">
-        <property name="max" value="120"/>
+        <property name="max" value="150"/>
```

Comment:
The line length was decided and implemented in another issue. The line length may not be changed.

```suggestion
        <property name="max" value="120"/>
        <property name="severity" value="error"/>
```